systemtest/intake_async_test.go

```diff
@@ -56,17 +56,17 @@ func TestIntakeAsync(t *testing.T) {
 		systemtest.CleanupElasticsearch(t)
 		srv := apmservertest.NewServerTB(t)
 
-		systemtest.SendBackendEventsAsyncPayload(t, srv.URL, `../testdata/intake-v2/errors.ndjson`)
+		systemtest.SendBackendEventsPayload(t, srv.URL, `../testdata/intake-v2/errors.ndjson`)
 		// Ensure the 5 errors are ingested.
 		systemtest.Elasticsearch.ExpectMinDocs(t, 5, "logs-apm.error-*", nil)
 
 		// Send a request with a lot of events (~1920) and expect it to be processed
 		// without any errors.
-		systemtest.SendBackendEventsAsyncPayload(t, srv.URL, `../testdata/intake-v2/heavy.ndjson`)
+		systemtest.SendBackendEventsPayload(t, srv.URL, `../testdata/intake-v2/heavy.ndjson`)
```

Comment:
When this is the root cause, shouldn't https://github.com/elastic/apm-server/pull/9268/files#diff-74763070977bc9b95333184e98c4a318a3dcf2f6f6a62ba88de993bc65c726c1R41 also need to be changed then; to avoid that the `503` is issued due to the async requests using up the available resources?

Reply:
`systemtest.SendBackendEventsPayload` will perform the Intake HTTP request synchronously, while `systemtest.SendBackendEventsAsyncPayload` will do that asynchronously.

The Queue is full errors are only thrown for async requests that can't be processed